etc/containerbuddy.json

```diff
@@ -0,0 +1,31 @@
+{
+  "consul": "{{ .CONSUL }}:8500",
+  "preStart": "/usr/local/bin/prestart.sh",
+  "services": [
+    {
+      "name": "wordpress",
+      "port": 80,
+      "publicIp": true,
```

Comment:
`publicIp` was deprecated in favor of specifying the interface priority list. See example in https://github.com/autopilotpattern/jenkins/blob/master/etc/containerbuddy.json#L15-L24